samples/golang/webfrontend/src/app/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"html/template"
+	"log"
+	"net/http"
+	"strconv"
+	"sync"
+)
+
+// ServerMessage is a representation of a message from server
+type ServerMessage struct {
```

Comment:
These become public by Pascal casing them, but there are no consumers. Since they are implementation details of this package, make them camel cased.